src/main/java/org/sagebionetworks/bridge/participantroster/DownloadParticipantRosterWorkerProcessor.java

```diff
@@ -234,19 +244,15 @@ void writeStudyParticipants(CSVWriter csvFileWriter, List<StudyParticipant> stud
 
         for (int i = 0; i < csvHeaders.length; i++) {
             JsonElement jsonValue = json.get(csvHeaders[i]);
-            String value = "";
+            StringBuilder value = new StringBuilder();
```

Comment:
I don't think we need to use a StringBuilder if we're just adding one string and then immediately calling .toString() on it.